business_requirement_deliverable/models/business.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+# © <YEAR(2015)>
+# <Elico Corp, contributor: Eric Caudal, Alex Duan, Xie XiaoPeng(S)>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+
+
+class BusinessRequirement(models.Model):
+    _inherit = "business.requirement"
+
+    deliverable_lines = fields.One2many(
+        comodel_name='business.deliverable.line',
+        inverse_name='business_requirement_id',
+        string='Deliverable Lines',
+        copy=False,
+        readonly=True,
+        states={'draft': [('readonly', False)]}
+    )
+    estimated_time_total = fields.Float(
```

Comment:
name should be: resource_time_total
should be a function based on the total time of the resource lines